### DISCUSSION TITLE
I can't execute commands

### DISCUSSION THREAD

**Discussion by 0marek** [Minestom/Minestom#2399] [Help]
HI, I followed the Kody Simpson minestom commands tutorial, but i cant execute any.

This is what im getting in the console:
`[Ms-TickScheduler] WARN net.minestom.server.listener.manager.PacketListenerManager - Packet class net.minestom.server.network.packet.client.play.ClientSignedCommandChatPacket does not have any default listener! (The issue comes from Minestom)`

Could someone help me with this please?

**Comment by FrederikHeinrich** [comment]
What Mods have you installed on your Client?

**Reply by 0marek** [reply]
Hi, none its a vanilla client, but i was using Gate as a proxy. Switched to velocity and the problem went away.

**Reply by FrederikHeinrich** [reply]
Can you please try to disable "forceKeyAuthentication" in the Gate Config? 
And try again?

**Reply by 0marek** [reply]
I tried it and it still doesn't work. I think I'm gonna keep using Velocity, but tysm for the help. Btw, sorry for the late reply, I didnt notice you replied.
